i3pystatus/core/desktop.py

```diff
@@ -55,4 +66,12 @@ def display(self):
             if self.timeout:
                 notification.set_timeout(self.timeout)
             notification.set_urgency(self.URGENCY_LUT[self.urgency])
-            return notification.show()
+            try:
+                return notification.show()
+            except Exception as exc:
```

Comment:
Yeah, I considered doing that but figured I'd restrict it to debug logging... however, this would require that someone passes a log_level param when instantiating the DesktopNotification, so it's probably better to use self.logger.exception() here.